diff --git a/CMakeLists.txt b/CMakeLists.txt
index 93c3570..f5cefd6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -57,6 +57,9 @@ install(FILES
 
 export(TARGETS spline
   FILE ${PACKAGE_NAME}Config.cmake)
+install(EXPORT ${PROJECT_NAME}-targets
+  FILE ${PROJECT_NAME}-config.cmake
+  DESTINATION lib/${PROJECT_NAME})
 
 # Install headers
 install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/include/spline.h"
